Modules/System/Azure Function/app.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id":  "8556c7d2-a594-4682-bd90-b10bfb176587",
+    "name":  "Connect to Azure Functions",
+    "publisher":  "Microsoft",
+    "brief":  "Simplify using Azure Functions",
+    "description":  "Support for writing AL code and connect to Azure Functions",
```

Comment:
"description":  "Support for writing AL code and connecting to Azure Functions",